tools/gyp/pylib/gyp/generator/msvs.py

```diff
@@ -2662,6 +2662,15 @@ def _GetMSBuildGlobalProperties(spec, guid, gyp_file_name):
     else:
       properties[0].append(['ApplicationType', 'Windows Store'])
 
+  msvs_windows_sdk_version = None
+  if msvs_windows_sdk_version == None and version.ShortName() == '2017':
+    vs2017_sdk = '10.0.14393.0'
+    vs2017_sdk = os.environ.get('vs2017_sdk', vs2017_sdk)
+    if vs2017_sdk:
```

Comment:
I don't know of a way to set an empty variable in `cmd`, but it's possible that the variable is set from another program that calls this. Something like `os.environ['vs2017_sdk'] = ''` in python or empty variable when using gitbash.